GNUmakefile

```diff
@@ -0,0 +1,92 @@
+
+default: install
+
+build:
+	go build -v ./...
+
+clean-generator-poc:
+	rm -f ./internal/sdk/poc/example/*_gen.go
+	rm -f ./internal/sdk/poc/example/*_gen_test.go
+
+clean-generator-%: ## Clean generated files for specified resource
+	rm -f ./internal/sdk/$**_gen.go
+	rm -f ./internal/sdk/$**_gen_*test.go
+
+docs:
+	go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate
```

Comment:
earlier this line was without generate at the end, why is it needed now?

Reply:
> The tfplugindocs CLI has two main commands, validate and generate (generate is the default). 
it can be removed, but I put it here for clarity